main/opensmtpd/APKBUILD

```diff
@@ -1,14 +1,15 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Contributor: Jonathan Curran <jonathan@curran.in>
+# Contributor: Shiz <hi@shiz.me>
 # Maintainer: Jonathan Curran <jonathan@curran.in>
 pkgname=opensmtpd
-pkgver=5.9.2p1
-pkgrel=3
+pkgver=6.0.2p1
+pkgrel=0
 pkgdesc="secure, reliable, lean, and easy-to configure SMTP server"
 url="http://www.opensmtpd.org"
 arch="all"
 license="ISC"
-depends=""
+depends="libasr fts libevent db libressl"
```

Comment:
I wasn't quite aware if that was simply to help the abuild developer -- good to know!